scripts/indexes.js

```diff
@@ -147,9 +147,13 @@ db.notifications.ensureIndex({ "sent_at": 1 })
 
 //======hourly_test_stats======//
 db.hourly_test_stats.createIndex({ "_id.date": 1 }, { expireAfterSeconds: 26 * 7 * 24 * 3600 })  // 26 weeks TTL
+db.hourly_test_stats.createIndex({ "_id.project": 1 }, { "_id.requester": 1}, {"_id.task_name": 1}, {"_id.date": 1})
```

Comment:
Is there a ticket open for these indexes?

Reply:
Yes, they are in the existing ticket.